.changesets/updates-to-error-level-instead-of-debug-level-for-better-visibility-of-critical-failures-.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: patch
+type: change
+---
+
+Use error level for displaying backtraces instead for better visibility of critical failures.
```

Comment:
Sorry to be annoying, but: error level what? Instead of what?
```suggestion
When an error occurs while initialising AppSignal or when running a probe, use error level logging to log the error's backtrace.
```